### PR TITLE
fus mtime sorting

### PR DESCRIPTION
**Pull request by heronhaye**

### CODE REVIEW

**Comment by heronhaye** [comment]
test fails unrelated

File: go/libkb/secret_store.go [hunk 1]
```diff
@@ -173,6 +174,29 @@ func GetConfiguredAccountsFromProvisionedUsernames(m MetaContext, s SecretStoreA
 		configuredAccounts = append(configuredAccounts, account)
 	}
 
+	logins, err := getLoginTimes(m)
```
Comment (jakob223) [inline go/libkb/secret_store.go]: can you rename this to `loginTimeMap` or something? sorting is slightly confusing to read as is

File: go/libkb/secret_store.go [hunk 2]
```diff
@@ -173,6 +174,29 @@ func GetConfiguredAccountsFromProvisionedUsernames(m MetaContext, s SecretStoreA
...
+		logins = make(loginTimes)
+	}
+
+	sort.Slice(configuredAccounts, func(i, j int) bool {
+		iUsername := configuredAccounts[i].Username
+		jUsername := configuredAccounts[j].Username
+		iTime, iOk := logins[NormalizedUsername(iUsername)]
+		jTime, jOk := logins[NormalizedUsername(jUsername)]
+		if !iOk && !jOk {
+			return strings.Compare(iUsername, jUsername) < 0
```
Comment (jakob223) [inline go/libkb/secret_store.go]: maybe prefer logged-in accounts in this case?

File: go/libkb/secret_store.go (hunk 1 above)
Comment (heronhaye) [reply go/libkb/secret_store.go]: ok laszlo

**Comment by heronhaye** [comment]
feedback fed back into the code, will merge after CI

**Pull request merged**